radio/src/gui/colorlcd/model_setup.cpp

```diff
@@ -1589,13 +1589,17 @@ void ModelSetupPage::build(FormWindow * window)
         if (j > 0 && (j % 6) == 0)
           centerGrid.nextLine();
 
-        new TextButton(group, centerGrid.getSlot(6, j % 6), getStringAtIndex(s, STR_RETA123, i),
-                       [=]() -> uint8_t {
-                           BFBIT_FLIP(g_model.beepANACenter, bfBit<BeepANACenter>(i));
-                           SET_DIRTY();
-                           return bfSingleBitGet<BeepANACenter>(g_model.beepANACenter, i);
-                       },
-                       OPAQUE | (bfSingleBitGet(g_model.beepANACenter, i) ? BUTTON_CHECKED : 0));
+        new TextButton(
+            group, centerGrid.getSlot(6, j % 6),
+            getStringAtIndex(s, STR_RETA123, i),
+            [=]() -> uint8_t {
+              BFBIT_FLIP(g_model.beepANACenter, bfBit<BeepANACenter>(i));
+              SET_DIRTY();
+              return (bfSingleBitGet<BeepANACenter>(g_model.beepANACenter, i) ? 1 : 0);
```

Comment:
Oh you have to be kidding me... that simple? 😆

Reply:
Embarrassing, isn't it? :-)

Reply:
Trees, meet forest 😁